### ISSUE TITLE
bleachbit 3.x - fedora 31 stable - ImportError: No module named gi

### ISSUE THREAD

**Issue by jcpf** [bleachbit/bleachbit#658]
fedora 31 doesn't have bleachbit 3.x yet
and is pushing python2 obsolete, naturally

your package available:
bleachbit-3.0-1.1.fc28.noarch.rpm

### **system description:**

```
$ bleachbit -v
scandir is not available, so falling back to slower os.walk()

BleachBit version 3.0
Copyright (C) 2008-2019 Andrew Ziem.  All rights reserved.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

$ uname -r
5.3.7-301.fc31.x86_64

$ cat /etc/fedora-release
Fedora release 31 (Thirty One)

$ which python2
/usr/bin/python2
```
### **error:**

```
$ bleachbit
Traceback (most recent call last):
  File "/usr/bin/bleachbit", line 38, in <module>
    import bleachbit.GUI
  File "/usr/share/bleachbit/GUI.py", line 34, in <module>
    import gi
ImportError: No module named gi
```

### **solution:**

package name
python2-gobject-devel-3.34.0-3.fc31.x86_64

`$ sudo dnf install python2-gobject-devel`

works

**Tanks for your effort**

**Comment by jcpf** [comment]
I also consider that it is not an isse. However, it is interesting to leave the solution notice for other users who encounter the same problem. I leave it in your hands to do what you see fit. Maybe close this notice and put a note in the readme.
thank you

**Comment by az0** [comment]
I would like to edit the .spec, so the .rpm works properly

**Comment by az0** [comment]
Would you mind removing `python2-gobject-devel` and `python2-gobject` packages and then testing this package? [bleachbit-3.0.1-2.1.noarch.rpm](
https://build.opensuse.org/package/binary/download/home:andrew_z/bleachbit/Fedora_28/x86_64/bleachbit-3.0.1-2.1.noarch.rpm)

I think it should point you to the needed dependencies. It actually depends on `python2-gobject`, not `python2-gobject-devel`.

**Comment by jcpf** [comment]
Following your directions

**Uninstall:**

```
bleachbit
python2-gobject
python2-gobject-devel
```

**Install:**

`bleachbit   noarch  3.0.1-2.1  @commandline 576 k
`

Automatically install dependencies:

```
python2-gobject        x86_64  3.34.0-3.fc31 fedora  17 k 
python2-gobject-base   x86_64  3.34.0-3.fc31 fedora 317 k
```


no longer requires:

`python2-gobject-devel  x86_64  3.34.0-3.fc31  fedora  25 k`

**Comment by jcpf** [comment]
If you think we can close the issue

**Comment by az0** [comment]
Thank you for verifying! Nice documentation

**Comment by az0** [comment]
I published this fix in [BleachBit 3.1.0 beta](https://www.bleachbit.org/news/bleachbit-31-beta). Perhaps you could test and write back here with any comments about how it addresses your issue.
